kfs-purap/src/main/java/edu/arizona/kfs/module/purap/PurapPropertyConstants.java

```diff
@@ -0,0 +1,5 @@
+package edu.arizona.kfs.module.purap;

```

Comment:
It doesn't look like  kfs-purap/src/main/java/edu/arizona/kfs/module/purap/PurapPropertyConstants.java is actually being used. Please remove.